packages/aztec.js/src/proof/joinSplit/helpers.js

```diff
@@ -55,6 +40,21 @@ helpers.generateCommitmentSet = async ({ kIn, kOut }) => {
     return { commitments, m: inputs.length };
 };
 
+// constructs an AZTEC commitment directly from the setup algorithm's trapdoor key.
+// Used for testing purposes only; we don't know the trapdoor key for the real deal.
+const generateFakeCommitment = async (k, trapdoor) => {
```

Comment:
Would it make sense to move this to a testing folder/file btw?

Reply:
This is not part of this PR. Will prepare a new one for the `abiEncoder` and `proof` submodules.